src/app/services/phoenix-channel.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Socket } from "phoenix_js"
+import { environment } from '../../environments/environment';
+import { Store } from '@ngrx/store';
+import { AppState } from '../reducers/index';
+import { storeFreeze } from 'ngrx-store-freeze';
+import { LoadCommentsSuccessAction } from '../actions/comment';
+
+@Injectable()
+export class PhoenixChannelService {
+
+  constructor(store: Store<AppState>) {
+    let socket = new Socket(
+      `${environment.socketEndpoint}/socket`,
+      {
+        params: {
+          token: localStorage.getItem('access_token')
+        },
+        transport: WebSocket,
+        logger: ((kind, msg, data) => { console.log(`${kind}: ${msg}`, data) })
+      },
+    );
+    socket.connect();
+
+    socket.onOpen(ev => console.log("OPEN", ev));
+    socket.onError(ev => console.log("ERROR", ev));
+    socket.onClose(e => console.log("CLOSE", e));
+
+    var chan = socket.channel("comments:lobby", {});
```

Comment:
IMO `chan` naming can be better here something like `comment_channel` or `commet_lobby` would be more readable.